docs/source/reference/comparison.rst

```diff
@@ -0,0 +1,178 @@
+.. _sky-compare:
+
+Comparing SkyPilot with other systems
+=====================================
+
+We are often asked "How does SkyPilot compare with XYZ?". Providing an unbiased and up-to date answer for such questions is not easy, especially when the differences may be qualitative.
+
+This page tries to provide a comparison of SkyPilot with other systems, focusing on the unique features of SkyPilot. We welcome feedback and contributions to this page.
+
+
+SkyPilot vs Vanilla Kubernetes
+------------------------------
+
+Kubernetes is a powerful system for managing containerized applications. :ref:`Using SkyPilot to access your Kubernetes cluster <kubernetes-overview>` boosts developer productivity and allows you to scale your infra beyond a single Kubernetes cluster.
+
+Faster developer velocity
+^^^^^^^^^^^^^^^^^^^^^^
+
+.. figure:: https://blog.skypilot.co/ai-on-kubernetes/images/k8s_vs_skypilot_iterative_v2.png
+    :align: center
+    :width: 95%
+    :alt: Iterative Development with Kubernetes vs SkyPilot
+
+    Iterative Development with Kubernetes requires tedious updates to Docker images and multiple steps to update the training run. With SkyPilot, all you need is one CLI (``sky launch``).
+
+SkyPilot provides faster iteration for interactive development. For example, a common workflow for AI engineers is to iteratively develop and train models by tweaking code and hyperparameters by observing the training runs.
+
+* **With Kubernetes, a single iteration is a multi-step process** involving building a Docker image, pushing it to a registry, updating the Kubernetes YAML and then deploying it.
+
+* **With SkyPilot, a single command (``sky launch``) takes care of everything.** Behind the scenes, SkyPilot provisions pods, installs all required dependencies, executes the job, returns logs and provides SSH and VSCode access to debug.
```

Comment:
Code formatting doesn't work while bold.

Reply:
Changed to using `:strong:`, markdown syntax wasn't playing nicely with spaces